conda/install.py

```diff
@@ -261,16 +261,31 @@ def rm_rf(path, max_retries=5, trash=True):
         # islink('/path/to/dead-link') is True.
         try:
             os.unlink(path)
+            return
         except (OSError, IOError):
+            if trash and move_path_to_trash(path):
+                return
             log.warn("Cannot remove, permission denied: {0}".format(path))
```

Comment:
No I think it's probably ok where it is.  But if something does end up slipping through, we're going to want more details about the exception and how it happens.  errno gives us quite a bit I think.  In python2 both IOError and OSError are EnvironmentError, which have an errno attribute (https://docs.python.org/2/library/errno.html).  It's a little different in python3, which is why we have to do (OSError, IOError) for the exception here instead of just EnvironmentError.